src/vuplay-dashjs.js

```diff
@@ -1,102 +1,57 @@
-(function () {
-    // Set your mpeg-DASH URL here.
-    var streamURL = "<your-stream-url>";
-    // Please login to https://admin.drm.technology to generate a VUDRM token.
-    var vudrmToken = "<your-vudrm-token>";
-
-    // Override two dash.js methods so that we can set the Widevine license request body.
-    var overrideKeySystemWidevine = function () {
+// Override two dash.js methods so that we can set the Widevine license request body.
+function VUDRMKeySystemWidevine(token) {
+    /* We are currying the widevine method to pass the vudrm token  */
+    return function() {
         return {
-            getInitData: function (cpData, kid) {
```

Comment:
will need to be tested with a dash manifest with the pssh in the manifest. I don't think it is needed but will need testing to be sure!

Reply:
@somerobmitchell , can you provide test content for that?

And in which context (browser/os) should it be tested?

Reply:
I have asked around if there is one available. If not you can create one...
http://docs.unified-streaming.com/documentation/vod/streaming.html#option-mpd-inline-drm

The browsers will be widevine with chrome/FF and PlayReady with Edge.

Reply:
Actually I would be very happy to see how to create drm'd content. But to avoid wasting too much time on that, a little would be very appriciated (from you or Perry) 😄.

Thank you for your help!